src/MICore/project.json

```diff
@@ -32,7 +34,8 @@
     "System.Threading.Thread": "4.0.0",
     "System.Threading.Timer": "4.0.1",
     "System.Xml.ReaderWriter": "4.0.11",
-    "System.Xml.XmlSerializer": "4.0.11"
+    "System.Xml.XmlSerializer": "4.0.11",
+    "Newtonsoft.Json": "9.0.1"
```

Comment:
I don't know which one you should pick, but here are the options:
1. Compile against a 6.0.x version. Looks like the latest is 6.0.8. You would likely want to compile against the 'portable-net45+wp80+win8+wpa81+aspnetcore50' version in this case. In this case you don't need any more work assuming this version actually has everything you need.
2. Compile against a 8.0.x version. Looks like the latest is 8.0.3. In this case you don't need to do anything for VS 2017, but you should fix up Install.cmd the same way we do for System.IO.FileSystem.Primitives.dll.
3. Keep compiling against this version, but update the various setup things for the new dependency.

If it was me, I would start with 1. If that works - great! If not, keep going till I found something that actually works.